mne/viz/epochs.py

```diff
@@ -441,6 +441,22 @@ def _prepare_epochs_image_im_data(epochs, ch_type, overlay_times, order,
             ts_args_]
 
 
+def _make_epochs_image_axis_grid(axes_dict=dict(), colorbar=False,
+                                 evoked=False, fig=None):
+    """Set up a dictionary of axes for epochs images."""
+    axes_dict["image"] = axes_dict.get("image", fig.add_subplot(111))
+    if any((colorbar, evoked)):
```

Comment:
Perhaps an over-engineering of `if colorbar or evoked`? I only point it out because you were giving @cbrnr so much grief :)

Reply:
You think it was low of me to accuse him of writing MATLABesque code?

Reply:
Also if I remember correctly, I originally wanted to write something much dumber, something like `if any([x is True for x in {colorbar, evoked}]):`, and then this one came up and I thought, ha, that's much better.

Reply:
On the contrary I was impressed :)

Reply:
The irony here is that this is exactly the opposite of my over-engineering (or vs. any)... I guess I will have to take a closer look at your code more often 😉!